jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V69__Remove_Column_State_From_Post_Table.sql

```diff
@@ -0,0 +1 @@
+alter table POST drop column STATE;
```

Comment:
Could you please also leave a comment for which task this is done and maybe why.